tests/check/check_signed_video_auth.c

```diff
@@ -495,7 +495,7 @@ START_TEST(remove_one_p_nalu)
   //
   // SI                ->   (valid) -> .P
   //  IPPSI            ->   (valid) -> .....P
-  //      IPPSI        -> (invalid) -> NNNNP (1 missed)
+  //      IPPSI        -> (invalid) -> MNNNNP (1 missed)
```

Comment:
Are you sure it is added before the I-frame. Then it belongs to the previous GOP. That does not look correct.